src/website/shared/migrations/0018_remove_github_null_on_maintainers.py

```diff
@@ -4,10 +4,10 @@
 
 
 def drop_improper_maintainers(apps, schema_editor):
-    Maintainer = apps.get_model("shared", "NixMaintainer")
+    maintainer = apps.get_model("shared", "NixMaintainer")
```

Comment:
Ah, I guess rather here, perform exclusion of `migrations/`.

Reply:
fixed.